planemo/options.py

```diff
@@ -1599,6 +1601,10 @@ def profile_database_options():
     )
 
 
+def test_index_option():
+    return planemo_option("--test_index", default=1, type=int, help="Select which test to run. Counting starts at 1")
```

Comment:
```suggestion
    return planemo_option("--test_index", default=1, type=int, help="Select which test to check. Counting starts at 1")
```

Also I'm wondering if `1` is a good default or if it would be better to re-check all tests by default?

Reply:
We can't, since one test creates one invocation

Reply:
Marius kindly explained to me that this is not about multiple assertions for the same workflow invocation, but when defining tests for multiple workflows in the same file.

Are you OK with the change in wording suggested above though?

Reply:
Yes, that's much better!